.ruff.toml

```diff
@@ -58,7 +58,6 @@ ignore = [
   "TRY003", # prevents custom exception messages not defined in exception itself.
   "ISC001", # single line implicit string concatenation. formatter recommends ignoring this.
   "PTH123", # use Path.open instead of open
-  "UP038",  # isinstance with | instead of ,
```

Comment:
I'm fine with removing this, and I think we may have already done so for jwst, but just curious - why was this done now? I don't see any `isinstance` calls in the PR

Reply:
The rule is no longer present in ruff. When I ran pre-commit for this PR ruff warned me that this rule no longer exists so I removed it.